### PR TITLE
Prepare txs response

### PR DESCRIPTION
**Pull request by borcherd**
updated prepareTxs response: this was one of the discussed painpoints when implementing the sdk. We should not return an ethers type, bc not everyone will work with ethers. We now return IPeanutUnsignedTransaction, which holds the values the integrator needs to create a transaction themselves (to, value, data and nonce). Also updated the signAndSubmitTx to work with the new types.

### CODE REVIEW

File: yarn.lock
Comment (Hugo0) [inline yarn.lock:1]: No yarn! pnpm! bad!

File: src/consts/interfaces.consts.ts [hunk 1]
```diff
@@ -44,8 +44,11 @@ export interface ICreatedPeanutLink {
 	txHash: string
 }
 
-export interface IPeanutUnsignedTransactions {
-	unsignedTxs: TransactionRequest // change this any type to correct type
+export interface IPeanutUnsignedTransaction {
+	to: string
+	nonce: number
+	data: string
+	value: BigInt
```
Comment (Hugo0) [inline src/consts/interfaces.consts.ts:51]: just a comment that transactions in the ethereum sense also have other various other fields (gas params, storage access params, chainid etcetc)

could potentially be slightly confusing, but I think this is a minor gripe. Good as is imo

**Review by Hugo0 — APPROVED** [review]
Plz remove yarn, everth else good!

File: yarn.lock
Comment (borcherd) [reply yarn.lock:1]: yarn is easier to type than pnpm, always make spelling mistakes when trying to type pnpm xd

File: src/consts/interfaces.consts.ts (hunk 1 above)
Comment (borcherd) [reply src/consts/interfaces.consts.ts:51]: Yes, there are a lot of other fields but we do not populate these fields, the ones that are defined here are the only ones that are populated.

**Pull request merged**